node_modules/gh-series/lib/api.js

```diff
@@ -201,18 +201,27 @@ var getSerie = module.exports.getSerie = function(ctx, id, includeOrgUnits, call
         return callback(validator.getFirstError());
     }
 
-    SeriesDAO.getSerie(id, includeOrgUnits, function(err, serie) {
+    SeriesDAO.getSerie(id, includeOrgUnits, function(err, series) {
         if (err) {
             return callback(err);
         }
 
-        // Ensure that the serie is either being requested from the app it belongs to
+        // Ensure that the series is either being requested from the app it belongs to
         // or from an authenticated user on the global admin
-        if (serie.AppId !== ctx.app.id && (!ctx.user || !ctx.user.isGlobalAdmin())) {
-            return callback({'code': 401, 'msg': 'This serie cannot be retrieved on this application'});
+        if (series.AppId !== ctx.app.id && (!ctx.user || !ctx.user.isGlobalAdmin())) {
+            return callback({'code': 401, 'msg': 'This series cannot be retrieved on this application'});
         }
 
-        return callback(null, serie);
+        // Check whether the current user can manage the series
+        SeriesAuthz.canManageSeries(ctx, series, function(err, canManage) {
+            if (err) {
+                return callback(err);
+            }
+
+            series.canManage = canManage;
```

Comment:
There is test coverage for this under the `verify authorization when getting a series` test.
I'll extend the `assertGetSerie` function so it does the canManage check.